modules/tensor_mechanics/tests/Time_integration/Newmark_time_integration/tests

```diff
@@ -0,0 +1,8 @@
+[tests]
```

Comment:
I would really prefer to have multiple tests in one directory, rather than multiple subdirectories with one test each. Then what is currently the subdirectory name would be the block names in the ```tests``` file. That way we would not end up with a lot of tests that just end up named ```*.test```. I'd furthermore prefer if we could keep the naming scheme somewhat clean and _not_ mix lower case and upper case directory names. Some file managers sort these separate which looks crummy.